doc/fd.1

```diff
@@ -255,6 +255,14 @@ as a normal search pattern.
 Exclude files/directories that match the given glob pattern.
 This overrides any other ignore logic.
 Multiple exclude patterns can be specified.
+This uses the same syntax as ignore files (i.e. gitignore) and behaves as if
+.I pattern
+was a line in the global ignore file.
+
+Note that for purposes of ignore rules, the current directory
```

Comment:
"Current directory" is kind of ambiguous.  It's really the *search path* that is treated as the root, not the cwd.

Reply:
hmmm, this is.... interesting. 

It looks like if there are multiple search paths, then it use the _first_ one  for all:

```
❯ fd
a/
a/other
a/test/
b/
b/a
b/test/
test/

❯ fd --exclude '/test' --search-path a
a/other

❯ fd --exclude '/test' --search-path b
b/a

❯ fd --exclude '/test' --search-path b --search-path b
b/a
b/a

❯ fd --exclude '/test' --search-path b --search-path a
a/other
a/test/
b/a

❯ fd --exclude '/test' --search-path a --search-path b
a/other
b/a
b/test/
```